app/pages/index.js

```diff
@@ -128,7 +130,7 @@ const Home = ({ randomContributors }) => {
 
             <div className="relative w-full mx-auto space-y-10 lg:space-y-20 xl:space-y-36 max-w-7xl">
               <h2 className="px-6 text-3xl font-semibold xl:text-5xl xl:w-1/2">
-                Everything You Need For Production Apps
+                Alles was man für "Production Apps" braucht
```

Comment:
```suggestion
                Alles was du für "Production Apps" braucht
```

Ich finde die Anrede mit `du` sehr gut, kommt persönlicher rüber